lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -29,33 +29,38 @@
           $mime_split = explode(':', $task->file);
           $task->file = $mime_split[0] . ':' . $mimetype . $mime_split[1];
         }
-        $values = [
-          'nid' => $task->campaign_id,
+
+        $user = user_load($task->drupal_id);
+
+        $data = [
+          'campaign_id' => $task->campaign_id,
           'campaign_run_id' => $task->campaign_run_id,
           'quantity' => $task->quantity,
           'why_participated' => $task->why_participated,
           'file' => $task->file,
           'caption' => $task->caption,
           'type' => $task->type,
-          'crop_x' => $task->crop_x,
-          'crop_y' => $task->crop_y,
-          'crop_width' => $task->crop_width,
-          'crop_height' => $task->crop_height,
-          'crop_rotate' => $task->crop_rotate,
         ];
+        // $data = dosomething_rogue_transform_reportback($task);
```

Comment:
oh right - I will delete that! Just had as a placeholder if we wanted to send to the helper function.